blocks/inspector-controls/radio-control/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { withInstanceId } from 'components';
+import { isEmpty } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import BaseControl from './../base-control';
+import './style.scss';
+
+function RadioControl( { label, selected, instanceId, onChange, options = [] } ) {
+	const id = 'inspector-radio-control-' + instanceId;
+	const onChangeValue = ( event ) => onChange( event.target.value );
+
+	return ! isEmpty( options ) && (
+		<BaseControl label={ label } id={ id }>
+			<div className="blocks-radio-control">
```

Comment:
Is the div necessary here? what if we pass className to the `BaseControl` component? (same for all other controls)